src/sentry/static/sentry/app/components/charts/components/types.tsx

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
@billyvg Does no file exist yet like this because we haven't done much typescripting with charts?

Reply:
Correct